components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1518,6 +1528,10 @@ double RewardsServiceImpl::GetDoubleOption(const std::string& name) const {
 std::string RewardsServiceImpl::GetStringOption(const std::string& name) const {
   DCHECK(!name.empty());
 
+  if (name == ledger::option::kPaymentServiceURL) {
```

Comment:
I'm certain this is not the right place to put this, but I don't even know why this method exists because kStringOptions is an empty map